### DISCUSSION TITLE
Is there any way to pick folder(s)?

### DISCUSSION THREAD

**Discussion by strangeways-dev** [yasirkula/UnityNativeFilePicker#33] [Q&A] [answered]
Is there any way to pick folder(s) and return the path(s)?
Cheers

**Comment by yasirkula** [comment, marked as answer]
There's a native code for this but it isn't isolated, it's connected to ExportMultipleFiles: https://github.com/yasirkula/UnityNativeFilePicker/blob/634be9b273404124e6f707a6228afe7c1c924815/.github/AAR%20Source%20(Android)/java/com/yasirkula/unity/NativeFilePickerExportFragment.java#L67-L71

There's a more isolated code in SimpleFileBrowser plugin but it uses Storage Access Framework, so System.IO API won't work with the returned directory path: https://github.com/yasirkula/UnitySimpleFileBrowser/blob/1e8a795d1569bcdfbab4d1983f1da9cd559c3741/Plugins/SimpleFileBrowser/Scripts/FileBrowser.cs#L1736

BTW picking folders isn't possible with any of my plugins on iOS.

**Reply by strangeways-dev** [reply]
Perfect, I will take a look.
Thank you!
